utility/src/main/java/org/oppia/android/util/gcsresource/BUILD.bazel

```diff
@@ -0,0 +1,31 @@
+"""
+Utilities to Provides GCS resource bucket names.
```

Comment:
Suggest: ``Utilities for interacting with GCS backend resources.``

It helps to document slightly more generically since this may contain other things in the future.

Reply:
Done